packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -222,7 +222,7 @@ class OverlayEntry implements Listenable {
   }
 
   @override
-  String toString() => '${describeIdentity(this)}(opaque: $opaque; maintainState: $maintainState)';
+  String toString() => '${describeIdentity(this)}(opaque: $opaque; maintainState: $maintainState)${_disposedByOwner ? "(DISPOSED)" : ""}';
```

Comment:
This may be a bit confused. I think it is possible that it is waiting to be disposed?

Reply:
`_disposedByOwner` invalidates the `OverlayEntry` immediately. I think the only operation allowed on a disposed OverlayEntry is `removeListener` (same as `ChangeNotifier`).